src/cmds/scripts/win_postinstall.py

```diff
@@ -0,0 +1,632 @@
+import sys
+import getopt
+import os
+import subprocess
+import socket
+from tempfile import mkstemp
+from shutil import move
+import time
+
+install_path = ''
+pbs_path = ''
+
+
+def replace(file_path, pattern, subst):
+    fh, abs_path = mkstemp()
+    with os.fdopen(fh, 'w') as new_file:
+        with open(file_path) as old_file:
+            for line in old_file:
+                new_file.write(line.replace(pattern, subst))
+    # Remove original file
+    os.remove(file_path)
+    # Move new file
+    move(abs_path, file_path)
+
+
+def ValidateCredentials(username, password):
+    if (username != '' and password != ''):
+        with open(os.devnull, 'w') as fp:
+            pbs_account_binary = os.path.join(
+                pbs_path, "exec", "bin", "pbs_account.exe")
+            theproc = subprocess.Popen(
+                [pbs_account_binary,
+                    "-c", "-s", "-a", username, "-p", password],
+                stdout=fp,
+                stderr=fp)
+            theproc.communicate()
+            if theproc.returncode > 0:
+                print "Invalid credentials"
+                sys.exit(3)
+            else:
+                print "Post install process started..."
+                print "User Validation Success"
+    else:
+        print "Usage error: \n post_install.py -u <username> -p <password>"
+        sys.exit(4)
+
+
+def CreatePbsConfFile():
+    pbsconffile = os.path.join(pbs_path, 'pbs.conf')
+    with open(pbsconffile, 'w') as fp:
+        fp.write("PBS_SERVER=" + socket.gethostname() + "\r\n")
```

Comment:
From python doc "Do not use os.linesep as a line terminator when writing files opened in text mode (the default); use a single '\n' instead, on all platforms"
link: https://docs.python.org/3/library/os.html#os.linesep